src/util/appInsights.ts

```diff
@@ -0,0 +1,30 @@
+import { initAppInsights } from "@pagopa/ts-commons/lib/appinsights";
+import * as ai from "applicationinsights";
+import * as packageJson from "../../package.json";
+import { getConfigOrThrow } from "./config";
+
+const config = getConfigOrThrow();
+
+export const getTelemetryClient = (): ai.TelemetryClient =>
```

Comment:
great! it's not mandatory, but what if we used `@pagopa/ts-commons/lib/appinsight`? for example https://github.com/pagopa/io-pagopa-proxy/blob/f9367c9af92a1ba2544a6f76e5bfbacb3fe224cd/src/utils/AIUtils.ts#L51